ui/lib/pki/addon/components/page/pki-key-details.ts

```diff
@@ -25,7 +25,6 @@ export default class PkiKeyDetails extends Component<Args> {
       this.flashMessages.success('Key deleted successfully.');
       this.router.transitionTo('vault.cluster.secrets.backend.pki.keys.index');
     } catch (error) {
-      this.args.key.rollbackAttributes();
```

Comment:
This was throwing an error, I think because the record is in a destroyed state. Not sure if this will cause issues but I'm not sure I've seen a rollback done in this manner before.